.circleci/config.yml

```diff
@@ -14,8 +14,12 @@ jobs:
       - run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.31.0
       - run: golangci-lint run
       - run: make
-      - run: curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | BINDIR=/home/circleci/.local/bin sh
-      - run: goreleaser release --skip-publish --snapshot
+      - run: cd /tmp && curl -L --remote-name https://github.com/goreleaser/goreleaser/releases/download/v1.8.3/goreleaser_Linux_x86_64.tar.gz
+      - run: cd /tmp && curl -L --remote-name https://github.com/goreleaser/goreleaser/releases/download/v1.8.3/checksums.txt
+      - run: cd /tmp && sha256sum --ignore-missing -c checksums.txt
+      - run: mkdir -p /home/circleci/.local/bin
+      - run: tar -C /home/circleci/.local/bin -xf /tmp/goreleaser_Linux_x86_64.tar.gz goreleaser
+      - run: goreleaser release --skip-publish --snapshot --debug
```

Comment:
sorry about the hacky method...

Reply:
It's IMO better than `curl | bash`. :joy_cat:

Reply:
sure! but I did not want to pack this into a script with less repetition, etc, just because I didn't want to deal with figuring out the CircleCI environment on top of trying to fix this.

I'll follow up with something that moves this to a script that's called from here